docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,42 +1,67 @@
+.How devfiles work
 
-Development tools consume devfiles and transform the definition into a cloud workspace composed of multiple containers and cloud-native resources. Also, users can create any number of identical workspaces from the same devfile. This way, different teams can use and share workspaces in various ways, including a devfile registry or together with the source code of a project.
+As a YAML file, devfiles simplify the development process. Devfiles tell you the steps you must take to get your application up and running from a source code to an actual product.When you download a devfile, it assesses your application development lifecycle, then provides you strategies and best practices for end-to-end application development. Devfiles do not force you to use new workflows nor tools, but rather, provide you instructions to automate your current, manual processes. For example, a devfile can take the repository hosting your application source code, build the code, run the application on your local container, then deploy the application to cloud-native containers. Unlike many development tools, devfiles can build, run, and deploy, automating many of your manual processes and simplifying your end-to-end development.
 
-A devfile includes:
+.Features and benefits of devfiles
 
-* Guidance about runtime images
+Devfiles provide guidelines and instructions for enhanced development processes. Devfiles uses the command line interface (CLI)  tool, odo, to execute the actions: build run, deploy. By using odo, devfiles provide consistent patterns of behavior for applications written in different languages and create workspaces composed of multiple containers. In these containers, you can make any number of identical workspaces from the same devfile. Creating multiple workspaces lets you share your devfile with different teams. Sharing a single devfile across multiple teams, ensures that each team has the same user experience and the same build, run, deploy behaviors.
+
+Leverage the following devfile features:
+
+* Guidance for using runtime images
 * Example code
 * Build and CI commands
 * Deployment options
 
-Benefits of devfiles:
+Use devfiles to help you with the following:
+
+* Reduce the gap between development and deployment
+* Have a registry for finding and hosting stacks
+* Produce consistent build and run behaviors
+* Allow easy devfile customization
 
-* Reducing the gap between development and deployment
-* Having a registry for finding and hosting devfiles
-* Producing consistent build and run behaviors
-* Allowing easy devfile customization
+For a visual of how simple a devfile is, see the following YAML:
```

Comment:
Good catch. I got caught up in "selling" devfiles :P What do you think of the revision? 

Also, do you think this example is necessary? 

I only want to keep this example if we think it actually adds something significant and meaningful to the module. Otherwise, I'm happy to remove it.